src/features/content-answers/ContentAnswersModalContent.tsx

```diff
@@ -1,38 +1,77 @@
 import React from 'react';
 
-import Answer from './Answer';
-import Question from './Question';
-import WelcomeMessage from './WelcomeMessage';
-
-import { QuestionType } from './ContentAnswersModal';
+import ContentAnswersModalChat from './ContentAnswersModalChat';
 // @ts-ignore: no ts definition
 // eslint-disable-next-line import/named
 import { User } from '../../../common/types/core';
 
-import './ContentAnswersModalContent.scss';
+export type QuestionType = {
+    prompt: string;
+    answer?: string;
+    createdAt?: string;
+};
 
 type Props = {
-    currentUser: User;
+    currentUser?: User;
     fileName: string;
     isLoading: boolean;
     questions: QuestionType[];
 };
 
 const ContentAnswersModalContent = ({ currentUser, fileName, isLoading, questions }: Props) => {
+    const contentRef = React.useRef<HTMLDivElement | null>(null);
+    const messagesEndRef = React.useRef<HTMLDivElement | null>(null);
+    const userScrolled = React.useRef(false);
```

Comment:
Where is userScrolled added as a ref to an element?

Reply:
I don't think that we need userScrolled here so I removed it. And I will push a new commit without it.

Reply:
when you use useRef, it doesnt need to ref an element, just a value that doesnt require rendering, so in this case its keeping track of the users position of the scrollbar so it can check if youre at the bottom or not


reference: https://react.dev/reference/react/useRef